resteasy-spring-boot-starter-test/pom.xml

```diff
@@ -196,15 +196,15 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-deploy-plugin</artifactId>
-                <version>2.8.2</version>
+<!--                <version>2.8.2</version>-->
```

Comment:
I'd remove this and rely on the parent. If you need to override it I'd use the property from the parent instead.

Reply:
will do!